src/components/Container/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import Heading from 'components/Heading';
+import Spinner from 'components/Spinner';
+import Error from 'components/Error';
+
+export const AsyncContainer = ({ loading, error, children }) => {
```

Comment:
but the async container is used by the Container (which is what we would use everywhere else).